src/components/report/OpenTypeFeatures.vue

```diff
@@ -232,10 +232,24 @@ export default {
 			};`;
 		},
 		getFeatureCSS(feature) {
-			return this.font.featureCSS(feature, {
-				value: 1,
-				format: "both",
-			});
+			// Type 3 lookups are non-boolean, deal with those if we got 'em
+			if (
+				this.featureChars[feature] &&
+				this.featureChars[feature]["lookups"]
```

Comment:
Could be:

```js
if (this.featureChars[feature]?.lookups) { /* … */ }
```

(I remember we had the conversation on consistent access of object properties)